README.md

```diff
@@ -140,6 +140,11 @@ Made with :heart:&nbsp;  by [Chandrika Deb](https://github.com/chandrikadeb7)
 * [https://www.pyimagesearch.com/](https://www.pyimagesearch.com/)
 * [https://www.tensorflow.org/tutorials/images/transfer_learning](https://www.tensorflow.org/tutorials/images/transfer_learning)
 
+## Our Contributors
```

Comment:
Add an emoji here to go with the other headings. Rest all good to merge!

Reply:
I have changed it.